### ISSUE TITLE
ODK support for managing mapping sets and cross-ontology bridges

### ISSUE THREAD

**Issue by gouttegd** [INCATools/ontology-development-kit#626]
The ODK should provide standardised workflows to facilitate:

* publishing one or several mapping sets (in SSSOM TSV format), whether those mapping sets are
  * manually maintained,
  * derived from pre-existing xrefs;
  * derived from another custom mechanism;
* synchronising mapping sets from foreign sources;
* publishing cross-ontology bridges, whether those bridges are
  * derived from pre-existing xrefs (as most Uberon/CL bridges currently are);
  * derived from a SSSOM mapping set (as the FBbt-to-Uberon/CL bridges currently are);
  * custom-generated (as the MBA-to-Uberon bridges will be).

A [short draft presentation](https://docs.google.com/presentation/d/1oppxvngpr62dOocLgMQf1JljdJyZDP9URfoXTxx99ww/edit?usp=sharing) illustrating the use cases, following a discussion with @matentzn and @shawntanzk.

**Comment by gouttegd** [comment]
Support for managing _mapping sets_ (first two tasks as outlined in the initial comment) has been implemented.

As for the support for managing _bridge files_ (third task: generating and publishing bridge files from either xrefs, SSSOM sets, or a custom process), I am no longer sure it belongs to the ODK.

The only ontology I have where managing bridge files is necessary is Uberon, and its bridging process is probably too complex to serve as a good template for generalisation into the ODK. I suspect that most ontologies either (1) don’t even need to produce bridge files or (2) have much simpler needs than Uberon.

**Comment by StroemPhi** [comment]
Adding my use case for cros-ontology bridges, as requested by @gouttegd [on the OBO-Slack](https://obo-communitygroup.slack.com/archives/C01BKKED8R2/p1756551485053609?thread_ts=1756495771.634859&cid=C01BKKED8R2).

We will have SSSOM mapping set TSVs from CHMO to AFO, Wikidata, REX & FIX living in CHMO's ODK repo in the `src/mappings` folder. I would like to build OWL cross-ontology bridges from these, that can be served as additional release artefacts, so they should be part of the release build pipeline. Maybe publishing a CHMO release that imports these bridges might also be of interest. 

My idea was that these bridges can then be used by data integration tools, like in the NFDI4Chem Search Service.

**Comment by matentzn** [comment]
Yes thanks, that is how I would have done it, just noting that not everyone might be super thrilled to use SSSOM but, hey, they have not asked it so far, they will be happy with what we give them!
